include/wx/listbase.h

```diff
@@ -400,6 +400,9 @@ class WXDLLIMPEXP_CORE wxListCtrlBase : public wxSystemThemedControl<wxControl>
     bool InReportView() const { return HasFlag(wxLC_REPORT); }
     bool IsVirtual() const { return HasFlag(wxLC_VIRTUAL); }
 
+    // Check if the item is visible
+    virtual bool IsVisible(long WXUNUSED(item)) const { return false; };
```

Comment:
```suggestion
    virtual bool IsVisible(long WXUNUSED(item)) const { return false; }
```
This would result in an error in gcc pedantic mode.

Reply:
Why did you resolve this without actually making the change?

Reply:
Sorry, please forget the previous comment, I've apparently looked at the wrong/outdated version.